docs/topics/db/aggregation.txt

```diff
@@ -26,7 +26,6 @@ used to track the inventory for a series of online bookstores:
 
     class Publisher(models.Model):
         name = models.CharField(max_length=300)
-        num_awards = models.IntegerField()
```

Comment:
This is used yet to test case and models(tests/aggregation). 
Is it ok for removing this field?